test/testdata/configs/config-v24.json

```
@@ -0,0 +1,106 @@
+{
+    "Version": 23,
```

```suggestion
    "Version": 24,
```

should this be `24`?

---

derp. yes.